packages/core/src/percy.js

```diff
@@ -365,6 +380,27 @@ export class Percy {
     if (this.testing && JSON.stringify(this.testing) === JSON.stringify({})) { return true; }
     return tokenType !== 'web';
   }
+
+  syncMode(options) {
+    let syncMode = false;
+    if (this.config?.snapshot?.sync) syncMode = true;
+    if (options?.sync) syncMode = true;
+    if (options?.sync === false) syncMode = false;
```

Comment:
This condition is breaking global config basically overrides if sync = true is there in global config and nothing is passed in screenshot command.

Reply:
SDK issue. fixed in SDK